docs/source/user/settings.rst

```diff
@@ -410,6 +410,26 @@ For example:
 Using the setting above will only allow phone numbers from the UK (``+44``)
 or Cameroon (``+237``).
 
+.. note::
+
+    This setting is applicable only for organizations
+    which have :ref:`enabled the SMS verification option
+    <openwisp_radius_sms_verification_enabled>`.
+
+``OPENWISP_RADIUS_SMS_REQUEST_TIMEOUT``
```

Comment:
Time out is not a good name for this.

I would change it to ``OPENWISP_RADIUS_SMS_COOLDOWN``.